client/classic/upe/index.js

```diff
@@ -468,6 +468,7 @@ jQuery( function ( $ ) {
 				},
 			} );
 			if ( error ) {
+				await api.updateFailedOrder( paymentIntentId, response.order_id );
```

Comment:
When testing on the Pay for Order page, got a response is not defined error. I think you can just use the `orderId` variable from above here though and it works as expected then.
```suggestion
				await api.updateFailedOrder( paymentIntentId, orderId );
```

Reply:
D'oh! Believe it or not, but I specifically designed this function with these separate parameters in mind, so that I'd be able to use it with `orderId` in `handleUPEOrderPay` as well...and then I forgot about it and just copy-pasted away. Nice catch, cheers!